husarion_ugv_manager/src/lights_manager_node.cpp

```diff
@@ -180,6 +190,11 @@ void LightsManagerNode::EStopCB(const BoolMsg::SharedPtr e_stop)
   lights_tree_manager_->GetBlackboard()->set<bool>("e_stop_state", e_stop->data);
 }
 
+void LightsManagerNode::JoyCB(const sensor_msgs::msg::Joy::SharedPtr joy)
+{
+  lights_tree_manager_->GetBlackboard()->set<bool>("drive_state", joy->buttons[4]);
```

Comment:
```suggestion
  lights_tree_manager_->GetBlackboard()->set<bool>("drive_state", joy->buttons[kDeadManButtonIndex]);
```
Can we make this const?